database/postgres/postgres.js

```diff
@@ -0,0 +1,88 @@
+/* eslint-disable camelcase */
+/* eslint-disable no-console */
+const { Client } = require('pg');
+const moment = require('moment');
+const config = require('./db_postgres_config');
+
+const client = new Client(config);
+
+(async () => {
+  await client.connect()
+    .then(() => console.log('database connected!'))
+    .catch((error) => console.log('error connecting to the database:', error));
+})();
+
+// get all reviews by hostel name, joined with authors and hostels
+const getReviewsByHostel = async (id) => {
+  const queryStr = `SELECT * FROM reviews, authors, hostels
+  WHERE reviews.hostel_id=${id}
+  AND reviews.author_id = authors.id
+  AND reviews.hostel_id = hostels.id`;
+
+  const res = await client.query(queryStr);
+  return res.rows;
+};
+
+// get all reviews by id
+const getReviewsById = async (id) => {
+  const queryStr = `SELECT * FROM reviews, authors, hostels
+  WHERE reviews.review_id=${id}
+  AND reviews.author_id = authors.id
+  AND reviews.hostel_id = hostels.id`;
+
+  const res = await client.query(queryStr);
+  return res.rows;
+};
+
+const createReview = async (body) => {
+  const {
+    hostel_id, author_id, description, security, location,
+    staff, atmosphere, cleanliness, facilities, value, total,
+  } = body;
+
+  // get the current timestamp and format it
+  const timestamp = Date.now();
+  let createdAt = new Date(timestamp);
+  createdAt = moment(createdAt).format('YYYY-MM-DD');
+
+  const queryStr = `INSERT INTO reviews (hostel_id, author_id, description, date, security, location, staff, atmosphere, cleanliness, facilities, value, total) VALUES (${hostel_id}, ${author_id}, '${description}', '${createdAt}', ${security}, ${location}, ${staff}, ${atmosphere}, ${cleanliness}, ${facilities}, ${value}, ${total})`;
```

Comment:
Line formatting issues are universal for all of us sharing the same VSCode configuration.